zerver/lib/event_queue.py

```diff
@@ -629,6 +679,7 @@ def receiver_is_idle(user_profile_id, realm_presences):
     return off_zulip or idle
 
 def process_message_event(event_template, users):
```

Comment:
`six.text_type` is `unicode` on python 2 and `str` on python 3. You should read more about this here - http://zulip.readthedocs.io/en/latest/mypy.html#annotating-strings.